### ISSUE TITLE
When going to Stable?

### ISSUE THREAD

**Issue by frehberg** [rust-lang-nursery/packed_simd#213]
Hello Team, may I ask, when std::simd will be merged into Rust-stable? Is there a roadmap or milestones?

Great work, Frank

**Comment by gnzlbg** [comment]
There is an RFC open (https://github.com/rust-lang/rfcs/pull/2366). The next step in the process is for that RFC to be merged, or rejected|postponed|etc.

**Comment by ralfbiedert** [comment]
Given that 2366 was closed and that the current version uses 10+ nightly features,  I guess there is no chance that a non-nightly version of `packed_simd` will be around any time soon?

**Comment by gnzlbg** [comment]
Many of the nightly features used here are not intended for stabilization, but `libcore` and `libstd` can use them, so this can easily be exposed through there using a single nightly feature, and we used to do that for a long time.

**Comment by bitshifter** [comment]
Sorry to comment on an old issue but I was about to raise a new one asking the same question. Is there a path to stabilizsation for `packed_simd` at the moment? Is it blocked by other rustc features in the pipeline like const generics? Are there any issues the community can assist with to help with a stabilization effort?

Thanks for your work, `packed_simd` looks great.

**Comment by gnzlbg** [comment]
>  Is it blocked by other rustc features in the pipeline like const generics?

Yes.

> Is there a path to stabilizsation for packed_simd at the moment?

SIMD was in the 2018 roadmap, but it wasn't in the roadmap in 2019. It's hard to justify investing too much time in this when there is no consensus that this is something worth pushing over the finish line. 

Maybe that will change after `const generics`, but it is very likely that this feature will require extensions over `const generics`.
